sbc_email/data/scan_letter_params.xml

```diff
@@ -0,0 +1,18 @@
+<openerp>
+	<data noupdate="1">
+		<record id="share_on_nas" model="ir.config_parameter">
+			<field name="key">sbc_email.share_on_nas</field>
+			<field name="value">Public</field>
+		</record>
+
+		<record id="scan_letter_imports" model="ir.config_parameter">
```

Comment:
It's used to save manul import.
Would you prefer that I use a "config import" just for that case or keep it in config?